notebooks/404-style-transfer-webcam/404-style-transfer.ipynb

```diff
@@ -0,0 +1,482 @@
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Select one of the styles: Mosaic, Rain Princess, Candy, Udnie, and Pointilism from the button below to do the style transfer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Option to select different styles    \n",
+    "styleButtons = ToggleButtons(\n",
+    "    options=['MOSAIC', 'RAIN-PRINCESS', 'CANDY', 'UDNIE', 'POINTILISM'],\n",
+    "    description=\"Click one of the styles you want to use for the style transfer\",\n",
+    "    disabled=False,\n",
+    "    style={'description_width': '300px'})\n",
+    "   \n",
+    "interactive(lambda option: print(option), option=styleButtons)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Model\n",
+    "\n",
+    "### Download the Model\n",
+    "\n",
+    "Model for the selected style will be downloaded to `model_path` if they have not been downloaded before. The selected model comes from the ONNX Model Zoo, which means it could be used with OpenVINO directly  but in this notebook, we like to show you how to use the Model Optimizer to convert ONNX to Intermediate Representation (OpenVINO IR)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Directory to download the model from ONNX model zoo\n",
+    "base_model_dir = \"model\"\n",
+    "base_url = \"https://github.com/onnx/models/raw/main/vision/style_transfer/fast_neural_style/model\"\n",
+    "\n",
+    "# Selected ONNX model will be downloaded in the path\n",
+    "model_path = Path(f\"{styleButtons.value.lower()}-9.onnx\")\n",
+    "\n",
+    "style_url = f\"{base_url}/{model_path}\"\n",
+    "utils.download_file(style_url, directory=base_model_dir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Convert ONNX Model to OpenVINO IR Format\n",
+    "\n",
+    "Use Model Optimizer to convert the ONNX model to OpenVINO IR with `FP16` precision. The models are saved to the current directory. With these options, it is not necessary to normalize input data before propagating it through the network.\n",
+    "Every conversion should take up to 1 minutes. If the model has been already converted, this step is skipped.\n",
+    "\n",
+    "For more information about Model Optimizer, see the [Model Optimizer Developer Guide](https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Construct the command for Model Optimizer.\n",
+    "precision = \"FP16\"\n",
+    "\n",
+    "mo_command = f\"\"\"mo\n",
```

Comment:
Have we considered using mo.convert here in python directly? @adrianboguszewski @AnishaUdayakumar

Reply:
This is a good idea. However, I know @zhuo-yoyowz wants to use this notebook this weekend and 2022.3 won't be released until next week. I suggest merging it as it is. Then @AnishaUdayakumar will open a new PR immediately providing the updates to MO.